twine-upload.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
```

Comment:
typo?
```suggestion
set -euo pipefail
```

Reply:
It's not a typo, `-E` is a valid option. See https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

I guess nothing here would be required at all, I mainly added it because it's part of my `.sh` file template.

Reply:
Ah, when I checked `man set` I didn't see it there. But not I look at `man bash` and it's explained on that page.
TIL.